runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -719,13 +719,13 @@ fn pure_deploy_bytes(ctx: &mut EstimatorContext) -> GasCost {
     let config_store = RuntimeConfigStore::new(None);
     let vm_config = config_store.get_config(PROTOCOL_VERSION).wasm_config.clone();
     let small_code = generate_data_only_contract(0, &vm_config);
-    let large_code = generate_data_only_contract(bytesize::mb(4u64) as usize, &vm_config);
+    let large_code = generate_data_only_contract(bytesize::kb(1500u64) as usize, &vm_config);
     let small_code_len = small_code.len();
     let large_code_len = large_code.len();
     let cost_empty = deploy_contract_cost(ctx, small_code, Some(b"main"));
-    let cost_4mb = deploy_contract_cost(ctx, large_code, Some(b"main"));
+    let cost_15mb = deploy_contract_cost(ctx, large_code, Some(b"main"));
```

Comment:
is this function for a test? otherwise not sure why it needs to specifically call out the limits, which are set in params files.

Reply:
> is this function for a test? otherwise not sure why it needs to specifically call out the limits, which are set in params files.

Ok I'll change it so that it uses the param file instead of a hardcoded value